ng-dev/pr/common/labels/action.ts

```diff
@@ -0,0 +1,23 @@
+import {createTypedObject, Label} from './base.js';
+
+interface ActionLabel extends Label {}
+
+export const actionLabels = createTypedObject<ActionLabel>()({
```

Comment:
Why not just assert the labels like this?

```suggestion
export const actionLabels: Record<string, ActionLabel> = {
```

Reply:
Having this style of typed object allows us to have specific keys while still enforcing that all of the values are of a specific type.

Reply:
But in either case you need to explicitly add extra code here. So using an explicit type assertion (like in my suggestion) enforces that the object conforms to the `Record` type too?

Reply:
Ahh, I didn't catch that you want the keys to explicit in the type. Makes sense